strongbox-configuration/src/main/java/org/carlspring/strongbox/yaml/repository/remote/RemoteRepositoryConfigurationDto.java

```diff
@@ -9,7 +9,7 @@
  * @author Pablo Tirado
  */
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-public abstract class MutableRemoteRepositoryConfiguration
+public abstract class RemoteRepositoryConfigurationDto
```

Comment:
:+1: I think this looks better! Thanks!

Reply:
yeah, this is actually was the reason of this naming refactoring, so all classes now follow same convention. This mess was really confusing.